src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -26,7 +25,7 @@ interface ThemeProviderThemeConfig extends Discard<ThemeConfig, 'colorScheme'> {
 
 interface ThemeProviderProps {
   /** Custom logos and colors provided to select components */
-  theme: ThemeProviderThemeConfig;
+  theme?: ThemeProviderThemeConfig;
```

Comment:
When would we not passing a theme into a theme provider make sense?

Reply:
If nothing is passed in then it returns the default CSS variables. What do you think about that assumption?

Reply:
I would have expected a theme without an config should be a non-operation:

```
<ThemeProvider theme={{colorScheme: 'dark'}}>
  <ThemeProvider>
    <Button>I would expect this button to still be in a dark scheme</Button>
  </ThemeProvider>
</ThemeProvider>
```

Forcing a color scheme of light seems to breakt that